pass/pass_linux.go

```diff
@@ -95,12 +79,38 @@ func runPass(stdinContent string, args ...string) (string, error) {
 	return string(result), nil
 }
 
+func isInitialized() bool {
+	if passInitialized {
+		return passInitialized
+	}
+	// In principle, we could just run `pass init`. However, pass has a bug
+	// where if gpg fails, it doesn't always exit 1. Additionally, pass
+	// uses gpg2, but gpg is the default, which may be confusing. So let's
+	// just explictily check that pass actually can store and retreive a
+	// password.
+	password := "pass is initialized"
+	name := path.Join(PASS_FOLDER, "docker-pass-initialized-check")
+
+	_, err := runPass(password, "insert", "-f", "-m", name)
+	if err != nil {
+		return false
+	}
+
+	stored, err := runPass("", "show", name)
+	passInitialized = err == nil && stored == password
```

Comment:
The error is silent here, maybe you should forward it to the caller?

Reply:
agree, would be cool to return a bool + err .